exonum/src/node/state.rs

```diff
@@ -586,7 +586,10 @@ impl State {
             }
             *known_round = round;
         }
-        let max_byzantine_count = self.validators().len() / 3;
+
+        // At max we can have (1/3) - 1 byzantine nodes.
```

Comment:
Nitpick: I don't quite understand the arithmetic in the comment, although the code is correct. Suppose `N` is the total number of nodes, `f` is the number of Byzantine nodes. By the BFT security assumptions, `N >= 3*f + 1`, i.e., `f <= (N - 1)/3` (which, in turn, is equivalent to `f <= (N + 2)/3 - 1`, which is used in the code).

Reply:
Well, I tried to follow official [documentation](https://exonum.com/doc/advanced/consensus/specification/) which tells, that we should have +2/3 non-Byzantine nodes (which is equivalent to -1/3 Byzantine ones). So here I just tried to simplify it as (1/3) - 1. Probably, that's not quite correct indeed.